### ISSUE TITLE
The self-learning AI controller for adaptive power beaming with fiber-array laser transmitter system. (arXiv:2204.05227v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#17830]
The self-learning AI controller for adaptive power beaming with fiber-array laser transmitter system. (arXiv:2204.05227v1 [eess.SY])<br>
https://ift.tt/E94F3fM<br>
<p>In this study we consider adaptive power beaming with fiber-array laser transmitter system in presence of atmospheric turbulence. For optimization of power transition through the atmosphere fiber-array is traditionally controlled by stochastic parallel gradient descent (SPGD) algorithm where control feedback is provided via radio frequency link by an optical-to-electrical power conversion sensor, attached to a cooperative target. The SPGD algorithm continuously and randomly perturbs voltages applied to fiber-array phase shifters and fiber tip positioners in order to maximize sensor signal, i.e. uses, so-called, "blind" optimization principle.</p>
<p>In opposite to this approach a perspective artificially intelligent (AI) control systems for synthesis of optimal control can utilize various pupil- or target-plane data available for the analysis including wavefront sensor data, photo-voltaic array (PVA) data, other optical or atmospheric parameters, and potentially can eliminate well-known drawbacks of SPGD-based controllers. In this study an optimal control is synthesized by a deep neural network (DNN) using target-plane PVA sensor data as its input. A DNN training is occurred online in sync with control system operation and is performed by applying of small perturbations to DNN's outputs. This approach does not require initial DNN's pre-training as well as guarantees optimization of system performance in time. All theoretical results are verified by numerical experiments.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
